s3/session.go

```diff
@@ -104,6 +104,17 @@ func getDefaultConfig(settings map[string]string) *aws.Config {
 
 	config = config.WithCredentials(newCredentials)
 
+	if logLevel, ok := settings[LogLevel]; ok {
+		config = config.WithLogLevel(func(s string) aws.LogLevelType {
+			switch s {
+			case "DEVEL":
```

Comment:
Maybe "DEBUG"?

Reply:
maybe, but we have similar setting in wal-g https://github.com/wal-g/wal-g/blob/98f4d56e3e453a8a572a4f29684377f2658c862c/tests_func/images/mongodb/config/wal-g-4.2.json#L22